Makefile

```diff
@@ -342,7 +342,6 @@ genjsonschema: genjsonschema-cleanup $(GOJSONSCHEMA)
 	@echo Modify jsonschema generated files.
 	sed -f ./config/jsonschema_patch.sed ${GENERATED_CONFIG} > ${GENERATED_CONFIG}.tmp
 	mv ${GENERATED_CONFIG}.tmp ${GENERATED_CONFIG}
-	$(MAKE) lint
```

Comment:
I don't follow. We don't run `generate` on `lint`, but we do run `lint` at the end of `genjsonschema`. Why is this changing now that we are running `genjsonschema` in our CI?

Reply:
Ah, I see the misunderstanding. I messed up my updates from the discussion above.

With my latest commit, `jsonschemagen` now runs alongside `generate`. We therefore don't want to run `lint`, because it's already being run in `precommit` and `ci`.